apps/emqx/test/emqx_takeover_SUITE.erl

```diff
@@ -188,6 +188,7 @@ t_takeover_willmsg(Config) ->
         {will_topic, WillTopic},
         {will_payload, <<"willpayload">>},
         {will_qos, 0}
```

Comment:
Nit: Only these three are `will` options, the `WillOpts` var name  at line 185 is misleading.